pub-critic/src/app/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,195 @@
+"use client";
+import { useEffect, useState } from "react";
+import classes from "./Filter.module.scss";
+import Dropdown from "components/Dropdown";
+import { Genre, Platform } from "api/GamesShared";
+import { FilterProps } from "api/GamesApi";
+import Slider from "react-slider";
+import FilterInput from "components/FilterInput";
+import Link from "next/link";
+import Switch from "components/Switch";
+
+interface Props {
+  filter?: (value: FilterProps) => void;
+  genres: Genre[];
+  platforms: Platform[];
+  searchParams?: {
+    search?: string;
+    genre?: number;
+    platform?: number;
+    metacritic?: string;
+    page?: number;
+    pageSize?: number;
+    ordering?: string;
+  };
+}
+
+const createRatingString = (min: number, max: number) => {
+  const values = [];
+  for (let i = min; i <= max; i++) {
+    values.push(i);
+  }
+  return values.toString();
+};
```

Comment:
this could be in the `app/utils/static` folder

Reply:
this one is not used at all so there is no reason to keep the function